src/main/java/de/tum/in/www1/artemis/domain/Achievement.java

```diff
@@ -0,0 +1,140 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@Entity
+@Table(name = "achievement")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Achievement implements Serializable {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "icon")
+    private String icon;
+
+    @Column(name = "rank")
+    private Integer rank;
```

Comment:
I'd rather use an enum here to standardize the ranks across the system. For example AchievementRanks with PLATIN, GOLD, SILVER, BRONZE. Then you can easily couple these values with your achievement definitions.

Reply:
That´s a good point actually, thanks 👍